scripts/get_distance_to_focal_set.py

```diff
@@ -0,0 +1,183 @@
+"""
+Mask initial bases from alignment FASTA
```

Comment:
Can we delete this old docstring or update it to reflect this script's function?

Reply:
oops...